controllers/errors.controllers.js

```diff
@@ -0,0 +1,22 @@
+
+exports.handleCustomErrors = (err,req,res,next)=>{
```

Comment:
`handleCustomErrors` and `handleSQLErrors` are not needed yet, so should be removed from this PR
It's best practise to only include code in a PR that meets the needs of the ticket 🕵️